sksparse/cholmod.pyx

```diff
@@ -145,6 +147,37 @@ cdef class Factor
 class CholmodError(Exception):
     pass
 
+class CholmodNotPositiveDefiniteError(CholmodError):
+    def __init__(self, message, column=None, factor=None):
+        super().__init__(message)
+        self.column = column
+        self.factor = factor
+
+class CholmodNotInstalledError(CholmodError):
+    def __init__(self, message=""):
+        message = "method not installed: {}".format(message)
```

Comment:
What does message look like?  An error code?
Let's be consistent with capitalization of the error message (compare with NotPositiveDefinite).  I think error messages tend to be capitalized(?).